components/dropdownlist/filter.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Filter
+page_title: DropDownList - Filter
+description: Filtering in the DropDownList for Blazor.
+slug: components/dropdownlist/filter
+tags: telerik,blazor,drop,down,list,dropdownlist,filter
+published: True
+position: 5
+---
+
+# DropDownList Filter
+
+The Filter in the DropDownList component is an input shown at the top of the dropdown popup, used for filtering the underlying datasource.
+
+To enable filtering, set the `Filterable` parameter to `true`.
+
+Filtering ignores casing and the default filter operator is `starts with`. Filtering looks in the `TextField`, and the filter is reset when the dropdown closes. You can choose a different operator through the `FilterOperator` parameter that takes a member of the `Telerik.Blazor.StringFilterOperator` enum.
+
+You can also implement custom (server) filtering and set a data source dynamically through the [`OnRead` event]({%slug components/dropdownlist/events%}#onread).
+
+>caption Filtering in the DropDownList
+
+````CSHTML
+@* Type something in the input to see items whose text contains only the typed string, for example "uct 2" *@
+
+<TelerikDropDownList Data="@Data"
+                     Filterable="true"
+                     FilterOperator="@filterOperator"
+                     @bind-Value="@SelectedValue" TextField="ProductName" ValueField="ProductId">
+</TelerikDropDownList>
+
+@code {
+    public List<Product> Data { get; set; }
+    public int? SelectedValue { get; set; }
+    StringFilterOperator filterOperator { get; set; } = StringFilterOperator.Contains;
```

Comment:
The filter operator is not necessary for the basic example, keep it only on the second example for choosing the operator and remove it from this sample.

Reply:
Done.